libs/langchain/langchain/llms/bedrock.py

```diff
@@ -184,7 +184,7 @@ def validate_environment(cls, values: Dict) -> Dict:
             if values["endpoint_url"]:
                 client_params["endpoint_url"] = values["endpoint_url"]
 
-            values["client"] = session.client("bedrock", **client_params)
```

Comment:
ooc any reason to make this user-configurable, or is bedrock-runtime always preferred?

Reply:
It should be "bedrock-runtime" always for Inference APIs (InvokeModel, InvokeModelWithResponseStream APIs)